ansible/roles/service/hippo_delivery/templates/sudoers.d/hippo_delivery.j2

```diff
@@ -0,0 +1,3 @@
+{{ hippo_delivery.user }} ALL= NOPASSWD: /bin/systemctl start {{ hippo_delivery.user }}
```

Comment:
Suggest a comment "The service name is by definition the same as the user name", as discussed.